docs/walkthroughs/using-the-bundled-source.md

```diff
@@ -17,8 +17,9 @@ And then look in the `node_modules` folder for the bundled `slate.js` file:
 node_modules/
   slate/
     dist/
-      slate.js
-      slate.min.js
+      umd/
```

Comment:
Let's change this to be not `dist/umd/` but instead just `umd/`, to match other React libraries. (Unless this is just a typo!)

Reply:
Hmm, currently the built source in the slate library is at `/dist/slate.js` and `/dist/slate.min.js`, so for the new output I used `/dist/es/slate.js`, `/dist.cjs/slate.js`, and `/dist/umd/slate.js` (slash `/dist/umd/slate.min.js`). So you want to drop the `/dist` and have three top-level output folders (`/es`, `/cjs`, and `/umd`)? (Doesn't matter a ton to me, I just thought it was somewhat customary to use a `/dist` folder, in addition to being a bit closer to what we currently ship)

Reply:
I agree, I much prefer `/dist` to `/umd`, I was just thinking of following React's lead. I think `react`, `react-dom` and `react-router` now are all using `/umd` for some reason. That said, I think for now let's go with:

```
dist/
  slate.js
  slate.min.js
lib/
  slate.js
  slate.es.js
```

Where `/dist` holds the UMD files, and `/lib` holds the CJS/ES6 files. This way I believe existing links to Slate's UMD files will remain unbroken.